libs/common/src/common/types/api.py

```diff
@@ -0,0 +1,33 @@
+from pydantic import BaseModel
+from typing import Optional, List
+
+
+class TalkRequestInputIntents(BaseModel):
+    intent: str
+    confidence: Optional[float] = None
+
+
+class TalkInput(BaseModel):
+    text: Optional[str] = None
+    intents: Optional[List[TalkRequestInputIntents]] = None
+    suggestion_id: Optional[str] = None
+
+
+class TalkRequest(BaseModel):
+    session_id: Optional[str] = None
+    input: TalkInput
+
+
+class TalkResponseOptionsItems(BaseModel):
+    label: str
+    input: Optional[TalkInput] = None
+
+
+class TalkResponseListItems(BaseModel):
+    text: Optional[str] = None
+    options: Optional[List[TalkResponseOptionsItems]] = None
+
+
+class TalkResponse(BaseModel):
+    response: Optional[List[TalkResponseListItems]] = None
+    session_id: str
```

Comment:
This would be better suited in the virtual-assistant package - I don't see any reason why this would be share by both services.

Reply:
Yeah, that makes sense. Done!